### PR TITLE
Subscribers search input is too small #16

### LINKED ISSUE #16: 🤔 [ISSUE] - Subscribers search input is too small
### Description

image

### Reproduction steps

```bash
1. Login
2. Click on 'Manage Subscribers > Subscribers'
```

### Logs

_No response_

### Browsers

_No response_

### OS

_No response_

### PR DESCRIPTION
**Pull request by adititripathi**
Search Input size increased

# Motivation

Bug fix to inrease the keyword size for input

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #16

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (Storybook)
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by adititripathi** [comment]
https://github.com/user-attachments/assets/b04aa697-af8f-4698-9ef3-cfbf70a8b9b2

File: frontend/src/components/subscribers/index.tsx [hunk 1]
```diff
@@ -172,15 +172,23 @@ export const Subscribers = () => {
...
         >
-          <FilterTextfield onChange={onSearch} fullWidth={true} />
+          <FilterTextfield
+            onChange={onSearch}
+            fullWidth={true}
+            sx={{
+              flexGrow: 1, // Allows the input to expand based on available space
+              flexShrink: 1, // Shrinks the input when space is limited (responsive)
+              flexBasis: "auto", // Ensures flexible behavior for the input size
+            }}
```
Comment (marrouchi) [inline frontend/src/components/subscribers/index.tsx:190]: I don't think we need this.

File: frontend/src/components/subscribers/index.tsx [hunk 2]
```diff
@@ -172,15 +172,23 @@ export const Subscribers = () => {
       <EditSubscriberDialog {...getDisplayDialogs(editDialogCtl)} />
       <PageHeader icon={FolderIcon} title={t("title.subscribers")}>
         <Grid
+          container
           justifyContent="flex-end"
           gap={1}
-          container
           alignItems="center"
-          flexShrink={0}
+          flexDirection="row"
```
Comment (marrouchi) [inline frontend/src/components/subscribers/index.tsx:179]: By default, it's "row". So no need to specify it.

**Review by marrouchi — COMMENTED** [review]
Thank you @adititripathi for your contribution 🙏

**Review by marrouchi — CHANGES_REQUESTED** [review]
Left you a couple of comments

**Comment by marrouchi** [comment]
Closed in favor of https://github.com/Hexastack/Hexabot/pull/59

**Pull request closed without merging**